### ISSUE TITLE
6.0.2: FLAKE8-check failure

### ISSUE THREAD

**Issue by mtelka** [jaraco/inflect#177]
`python -m pytest` fails for `inflect 6.0.2` with the following error (tested with `flake8-noqa 1.3.0`):
```
=================================== FAILURES ===================================
_________________________________ FLAKE8-check _________________________________
$(BUILD_DIR)/inflect/__init__.py:3631:55: NQA102 "# noqa: C901" has no matching violations
$(BUILD_DIR)/inflect/__init__.py:3790:69: NQA102 "# noqa: C901" has no matching violations
```

**Comment by jaraco** [comment]
I believe this issue was partially fixed incidentally in c81b81b053581f89c27ad20e604eb39d3a204de0.

This project doesn't use flake8-noqa, so the test suites continue to pass.

**Comment by jaraco** [comment]
Looking at https://github.com/jaraco/inflect/actions/runs/4624325191/jobs/8179045229, I see that the dual C901 noqa is required because flake8 requires the directive in different places on different Pythons, meaning this project is incompatible with flake8-noqa. I'm open to other options, but for now, I don't think there's a solution here except not to use flake8-noqa.
